cmake/CMakeLists.txt

```diff
@@ -1483,6 +1474,26 @@ install(
   DESTINATION ${CMAKE_INSTALL_SYSCONFDIR}/profile.d
 )
 
+###############################################################################
+# Install LAMMPS module into site-packages folder
+# Only available, if a shared library is built
+###############################################################################
+if(BUILD_LIB AND BUILD_SHARED_LIBS)
+  find_package(PythonInterp)
+  add_custom_target(
+    install-python
+    ${PYTHON_EXECUTABLE} install.py -v ${LAMMPS_SOURCE_DIR}/version.h
+    -m ${CMAKE_CURRENT_SOURCE_DIR}/../python/lammps.py
+    -l ${CMAKE_BINARY_DIR}/liblammps${CMAKE_SHARED_LIBRARY_SUFFIX}
```

Comment:
Could we add `-d ${PYTHON_INSTDIR}` here?

Reply:
With the recent change, should we are `-p` here? CMake's default prefix already doesn't require root.

Reply:
I am OK with any change, that keeps the conventional build and cmake builds with default settings consistent. E.g. on MacOSX the user site-packages folder for python is "$HOME/Library/Python/#.#/lib/python/site-packages". Does `${PYTHON_INSTDIR}` by default point there? While on Linux it would have to point to "$HOME/.local/lib/python#.#/site-packages/" by default.

Reply:
It runs `import distutils.sysconfig as cg; print(cg.get_python_lib(1,0,prefix='${CMAKE_INSTALL_PREFIX}))` to determine the library dir.